test/e2e/flask/multi-srp/common-multi-srp.ts

```diff
@@ -34,3 +43,16 @@ export async function withMultiSrp(
     },
   );
 }
+
+export async function mockActiveNetworks(mockServer: Mockttp) {
```

Comment:
every time we open the account for adding a new srp, a request is sent to this endpoint -- we should mock it.